department-of-veterans-affairs/next-build
# skip link added

## Description
Relates to #[15512](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15512). (or closes?)

Adds a skip link to all pages, only visible on focus and links to the first h1 or the content div

Changes:
Adds skip link to the wrapper component
Adds a div to the story listing template with a `#content` id, as it didnt have one previously

## Testing done
Local visual testing

## Screenshots
<img width="1395" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/fc8bf7fa-ec50-4145-aed5-f3e4050e299e">


## QA steps
- Navigate to a a story listing or news story page
- tab to first element above footer
- verify enter/clicking on the 'skip to content' link sets focus and scrolls to content as you would expect

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
